x-pack/plugins/security_solution/server/lib/detection_engine/rule_management/logic/exceptions/validate_rule_default_exception_list.ts

```diff
@@ -27,9 +27,11 @@ export class ExceptionListError extends Error {
 export const validateRuleDefaultExceptionList = async ({
   exceptionsList,
   rulesClient,
+  ruleRuleId,
   ruleId,
```

Comment:
Nit: we could improve the function interface a little bit by combining the `id` and `ruleId` params in an object and making them optional. For example:

```ts
interface ExceptionValidationParams {
  exceptionsList: ListArray | undefined;
  rulesClient: RulesClient; 
  rule?: {
    id?: string;
    ruleId?: string;
  };
}
```